runtime/devnet/src/config/api/versioning.rs

```diff
@@ -103,17 +103,9 @@ struct V0Error(pop_primitives::v0::Error);
 impl From<DispatchError> for V0Error {
 	fn from(error: DispatchError) -> Self {
```

Comment:
Preferably this should just be:
```suggestion
	fn from(error: DispatchError) -> Self {
	    Self(error.into())
	}
```

Makes me wonder whether to remove that whole `DecodingFailed` variant as the developer should never receive this error anyway @evilrobot-01 .